pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -145,7 +146,7 @@ func (s *Server) initDNSCerts(hostname, customHost, namespace string) error {
 		}
 		log.Infof("Generating istiod-signed cert for %v:\n %s", names, certChain)
 
-		signingKeyFile := path.Join(LocalCertDir.Get(), "ca-key.pem")
+		signingKeyFile := path.Join(LocalCertDir.Get(), ca.CAPrivateKeyID)
```

Comment:
"ID" feels misleading here. Its not really an ID but a filename?

Reply:
I agree, would making another const with the same value, and using correct semantics when applicable (i.e. file name) make sense to you? Something like: 
`const (`
    `CaCertID string "ca-cert.pem"`
    `CaCertFile`
`)`

Reply:
> When is ID every appropriate?

When building CASecret I imagine?